src/components/schema/address.tsx

```diff
@@ -0,0 +1,33 @@
+export type Address = { // reused in other components too, maybe have this type in somewhere central
```

Comment:
Don't you want the schema to live alongside the other component lib repo you all are working on?